pydoctor/driver.py

```diff
@@ -489,6 +489,12 @@ def p(msg: str) -> None:
                 for fn in sorted(system.docstring_syntax_errors):
                     p('    '+fn)
 
+            if len(system.root_names) == 1:
+                # If there is just a single root module it is written to index.html to produce nicer URLs.
+                # To not break old links we also create a symlink from the full module name to the index.html
+                # file. This is also good for consistency: every module is accessible by <full module name>.html
+                (build_directory / (list(system.root_names)[0] + '.html')).symlink_to('index.html')
```

Comment:
I just added a check to create a symlink so that old links outside of pydoctor don't break.